website/content/gateway/api_reference/resources/debits/recipient_information.md

```diff
@@ -28,4 +28,10 @@ Example: "Doe Jane A." (last name, first name, optional middle initial).
 {{% regex_optional %}}Required if the merchant account's Business Application Identifier (BAI) is Funds Disbursement (FD).{{% /regex_optional %}}
 {{% /description_details %}}
 
+{{% description_term %}}recipient[purpose_of_payment] {{% regex %}}[A-Z]{6}{{% /regex %}}{{% /description_term %}}
```

Comment:
Purpose of Payment is not related to recipient data so it does not make sense to have it in the `recipient` dictionary.

AFAICT, we should the same as for credits, i.e. have it on the level as e.g. `text_on_statement`. Because the Debits section currently only refers to the parameters for an authorization, we need to add a Parameters section to debits, stating that these parameters are in addition to the parameters supported for an authorization.